modin/pandas/dataframe.py

```diff
@@ -3121,8 +3121,12 @@ def __contains__(self, key):
     def __round__(self, decimals=0):
         return self._default_to_pandas(pandas.DataFrame.__round__, decimals=decimals)
 
-    def __setstate__(self, state):
-        return self._default_to_pandas(pandas.DataFrame.__setstate__, state)
+    @classmethod
+    def __inflate__(self, pandas_df):
+        return from_pandas(pandas_df)
```

Comment:
This seems like it will have serious performance issues, is this the only way?

Reply:
I don't think it's the only way, my original implementation just gave away `self._query_compiler` as a thing to pickle and then reconstructed `DataFrame` from it. The reason I chose this particular one is I wanted to make sure the ending pickle is persistent enough to be storable on disk and readable in a separate process run, for which I'm not sure storing Ray objects or whatever is what we need.

I'm also keeping in mind my goal of enabling local-remote context talk, and I don't think I can pickle local Ray object and unpickle it remotely.

Maybe we need a switch of "slow but storable" and "fast but works in one session only"?